src/ui/server/controllers/insurance/policy/map-submitted-data/loss-payee/index.ts

```diff
@@ -19,6 +20,20 @@ const mapSubmittedData = (formBody: RequestBody): object => {
     delete populatedData[IS_APPOINTED];
   }
 
+  if (objectHasProperty(populatedData, LOCATION)) {
+    if (populatedData[LOCATION] === IS_LOCATED_IN_UK) {
+      populatedData[IS_LOCATED_IN_UK] = true;
+      populatedData[IS_LOCATED_INTERNATIONALLY] = false;
+      delete populatedData[LOCATION];
+    }
+
+    if (populatedData[LOCATION] === IS_LOCATED_INTERNATIONALLY) {
+      populatedData[IS_LOCATED_INTERNATIONALLY] = true;
+      populatedData[IS_LOCATED_IN_UK] = false;
+      delete populatedData[LOCATION];
```

Comment:
Could have `delete populatedData[LOCATION];` in just one place at the end of the parent `objectHasProperty` if statement